### PR TITLE
fetch and display new line/route info with correct tz

### PR DESCRIPTION
**Pull request by mjdavidson**

### CODE REVIEW

File: js/data/LineData.js [hunk 1]
```diff
@@ -19,9 +20,10 @@ class LineData {
     }
     const line = encodeURIComponent(this.line_id)
     const agency = encodeURIComponent(this.agency_id)
+    const route = encodeURIComponent(this.route_id)
```
Comment (consindo) [inline js/data/LineData.js]: What if there's no route_id specified?

File: js/views/lines/LineList.jsx [hunk 2]
```diff
@@ -98,31 +99,20 @@ class LineList extends React.Component {
...
-    const { operators, friendlyNumbers } = this.state
     e.preventDefault()
     const url = [
       '',
       'l',
       match.params.region,
-      operators[link],
-      friendlyNumbers[link] || link,
+      link.agencyId,
+      link.routeId,
```
Comment (consindo) [inline js/views/lines/LineList.jsx]: Don't think this URL is currect

File: js/views/shell/Content.jsx [hunk 3]
```diff
@@ -86,7 +86,7 @@ class Content extends React.Component {
           <Route path="/l/:region" exact render={wrapFn(LineList)} />
           <Route path="/l/:region/all" exact render={wrapFn(AllLines)} />
           <Route
-            path="/l/:region/:agency_id/:line_id"
+            path="/l/:region/:agency_id/:route_id/:line_id"
```
Comment (consindo) [inline js/views/shell/Content.jsx]: Should be /l/:region/:agency_id/:line_id?route_id=whatever

**Review by consindo — CHANGES_REQUESTED** [review]
Looks pretty good, but move the route_id to a query parameter.

**Comment by mjdavidson** [comment]
@consindo sorry don't quite understand your aversion to route_id preference
https://github.com/dymajo/waka-server/blob/master/src/waka-worker/lines/regions/generic.ts#L12..L31
https://github.com/dymajo/waka-server/blob/master/src/waka-worker/stops/station.ts#L359..L377
all props needed to perform fetch by route_id are already supplied

**Comment by mjdavidson** [comment]
fine.

**Review by consindo — APPROVED** [review]

**Pull request merged**